scripts/build-glean-metadata.py

```diff
@@ -9,7 +9,7 @@
 import stringcase
 
 OUTPUT_DIRECTORY = os.path.join("public", "data")
-ANNOTATIONS_URL = "https://mozilla.github.io/glean-annotations/api.json"
+ANNOTATIONS_URL = "https://deploy-preview-8--blissful-roentgen-ca62bb.netlify.app/api.json"
```

Comment:
This is just for testing, it should be reverted before this lands.

Reply:
Maybe something to to pull from the environment via `.env`, if it's a common enough workflow.

Reply:
I'm not sure -- in this case I did it so people could preview the behaviour inside the deploy preview here. An environment variable wouldn't solve that case. 

It's worth bearing in mind though, I can see this being useful in some other cases. If this comes up again I'll definitely add this feature.